srcs/compose/compose/en_US_UTF_8_Compose.pre

```diff
@@ -599,8 +599,10 @@ XCOMM Latin Extended-A
 <Multi_key> <c> <apostrophe>		: "ć"	U0107 # LATIN SMALL LETTER C WITH ACUTE
 <dead_circumflex> <C>			: "Ĉ"	U0108 # LATIN CAPITAL LETTER C WITH CIRCUMFLEX
 <Multi_key> <asciicircum> <C>		: "Ĉ"	U0108 # LATIN CAPITAL LETTER C WITH CIRCUMFLEX
+<Multi_key> <C> <asciicircum>		: "Ĉ"	U0108 # LATIN CAPITAL LETTER C WITH CIRCUMFLEX
```

Comment:
This file is copied from an other project (from X11) and is not meant to be modified.
You should add the new compose sequences to this file instead: https://github.com/Julow/Unexpected-Keyboard/blob/master/srcs/compose/compose/extra.json